js/src/personal/amanda/Amanda.module.css

```diff
@@ -1,16 +1,18 @@
 .landingPage {
-    width: 100vw;
     height: 100vh;
-
+    background-color: light-dark(#E0FBFC, #3E4C5E);
```

Comment:
It'll likely be good to put these into your theme. We can discuss that afterwards.